tests/operators_test.py

```diff
@@ -17,7 +17,7 @@ def unknown_handler(self, name, args):
         return 42, SExp.to(b'foobar')
 
     def test_unknown_op(self):
-        self.assertRaises(EvalError, lambda: OPERATOR_LOOKUP(b'unknown-op', SExp.to(1337)))
+        self.assertRaises(EvalError, lambda: OPERATOR_LOOKUP(b'1337', SExp.to(1337)))
```

Comment:
heh, I meant to change this to be an odd number, but it's still an ascii "7", not a byte 7. Still appears to be an odd number though